extension/src/paymentHandler/manual-capture.handler.js

```diff
@@ -43,6 +43,12 @@ async function execute(paymentObject) {
         response.pspReference
       )
     )
+    actions.push(
+      pU.createChangeTransactionTimestampAction(
+        chargeInitialTransaction.id,
+        paymentObject.createdAt
```

Comment:
same as above

Reply:
Undo the timestamp assignment in extension module.
https://github.com/commercetools/commercetools-adyen-integration/pull/826/commits/9e35ab7f3f1285abb7119ef3a73d820293b3fba2